iOS_SDK/OneSignalSDK/Source/OSMessagingController.m

```diff
@@ -531,11 +533,35 @@ - (void)messageViewDidSelectAction:(OSInAppMessage *)message withAction:(OSInApp
     if (self.actionClickBlock)
         self.actionClickBlock(action);
     
+    if (message.isPreview) {
+        [self logFeaturesNonPreviewIAM:message withAction:action];
+        return;
+    }
+
+    // The following features are for non preview IAM
+    // Make sure no click, outcome, tag tracking is performed for IAM previews
     [self sendClickRESTCall:message withAction:action];
     [self sendTagCallWithAction:action];
     [self sendOutcomes:action.outcomes];
 }
 
+/*
+* Show the developer what will happen with a non IAM preview
+ */
+- (void)logFeaturesNonPreviewIAM:(OSInAppMessage *)message withAction:(OSInAppMessageAction *)action {
```

Comment:
yay this is perfect! lest rename it though from `logFeaturesNonPreviewIAM` to 
`processPreviewIam` or
`processPreviewInAppMessage`